INN/Largo
# Fix 1346, add missing taxonomy term labels trunk

## Changes

This pull request makes the following changes:

- Add missing taxonomy labels for prominence, series, and post-type. Fixes #1346

## Why

For #1346, referred as related by #1776

## Testing/Questions

Features that this PR affects:

- Custom Taxonomies


Questions that need to be answered before merging:


Steps to test this PR:

1. Check out branch
1. `wp taxonomy get --field=labels prominence`
1. Verify labels do *not* include the default category or tag labels for any key.
1. `wp taxonomy get --field=labels post-type`
1. Verify labels do *not* include the default category or tag labels for any key.
1. `wp taxonomy get --field=labels series`
1. Verify labels do *not* include the default category or tag labels for any key.

## Additional information

INN Member/Labs Client requesting: (if applicable)

- [x] Contributor would like to be mentioned in the release notes as: Chris Harmoney (Cornershop Creative)
